src/technochat/http/chat.go

```diff
@@ -92,9 +90,8 @@ func (s *Server) chatConnect(w http.ResponseWriter, r *http.Request) {
 		msg := chat.WSMessage{}
```

Comment:
should be in a separate goroutine

Reply:
why ?
to end request hendler? but request is upgraded into WS. this is good for now.

Reply:
will look more consistent